package-lock.json

```diff
@@ -6077,6 +6077,10 @@
       "integrity": "sha1-TkMekrEalzFjaqH5yNHMvP2reN8=",
       "dev": true
     },
+    "isosurface": {
+      "version": "git://github.com/archmoj/isosurface.git#e2c41594417a2ba363bb430c7d38adfe18312fc5",
```

Comment:
We should move that code over to `gl-isosurface`. That way it will be under the `gl-vis` organisation which all of us have access to.

Reply:
Oops, I didn't notice that `isosurface` repo is a fork of https://github.com/mikolalysenko/isosurface - we should try to get your branch merge into Mikola's official repo. My apologies.